packages/netlify-cms-backend-test/README.md

```diff
@@ -1,11 +1,20 @@
-# Docs coming soon!
+# Backend Test
 
-Netlify CMS was recently converted from a single npm package to a "monorepo" of over 20 packages.
-That's over 20 Readme's! We haven't created one for this package yet, but we will soon.
+This is `in-memory` file manager.
```

Comment:
The backend behind https://cms-demo.netlify.com/.
Used for demo purposes only

Reply:
fixed